docs/cli-help.txt

```diff
@@ -22,14 +22,11 @@ optional arguments:
   -v, --verbose         This flag can be used multiple times to incrementally
                         increase the verbosity of output, e.g. -vvv. (default:
                         -2)
-  -t, --text            Use the text output instead of the curses UI.
```

Comment:
They are rarely urgent, however, they do offer a better experience for the user. Why should we tie our hands and force ourselves to sit on these changes until we do a release?

And while I wouldn't call them urgent, there are definitely times we want to make changes outside of a release. For example, the stuff that I've been working on involves doc changes and needs to go live almost immediately so it is up before super secret project that I'm not sure I'm allowed to talk about publicly involving encryption, pipes, and Noah the hacker is released. Another example of changes like this is changes in OS packaging such as when a new version of a distro is released.